src/credentials/FabricTable.cpp

```diff
@@ -282,12 +282,7 @@ CHIP_ERROR FabricInfo::GeneratePeerId(FabricId fabricId, NodeId nodeId, PeerId *
         rootPubkey = rootPubkeySpan;
     }
 
-    ChipLogDetail(Inet, "Generating compressed fabric ID using uncompressed fabric ID 0x" ChipLogFormatX64 " and root pubkey",
-                  ChipLogValueX64(fabricId));
-    ChipLogByteSpan(Inet, ByteSpan(rootPubkey.ConstBytes(), rootPubkey.Length()));
     ReturnErrorOnFailure(GenerateCompressedFabricId(rootPubkey, fabricId, compressedFabricIdSpan));
-    ChipLogDetail(Inet, "Generated compressed fabric ID");
-    ChipLogByteSpan(Inet, compressedFabricIdSpan);
```

Comment:
Because these logs are very very chatty and serve no useful purpose, and they were filling-up the areas of logs where I needed to understand what's happening.

Reply:
OK, sounds good.